dts/bindings/pwm/infineon,cat1-pwm.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2024 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Infineon Cat1 PWM
+  Each PWM group Cat1 is assigned to a Zephyr device. Refer to the Infineon PSoC6 reference
+  manual (Section Port I/O functions) for the group/channel mapping to a specific port-pin on
+  the board.  For example on the cy8cproto_062_4343w P10.0 is mapped to adc0,channel0 and
+  P10.1 is mapped to adc0,channel1.
```

Comment:
need to update description "For example on the cy8cproto_062_4343w P10.0 is mapped to adc0,channel0 and
  P10.1 is mapped to adc0,channel1."